#No request
---
geometry_msgs/PoseStamped obj_pose